components/driver/sdmmc_transaction.c

```diff
@@ -153,7 +154,11 @@ esp_err_t sdmmc_host_do_transaction(int slot, sdmmc_command_t* cmdinfo)
     cmdinfo->error = ESP_OK;
     sdmmc_req_state_t state = SDMMC_SENDING_CMD;
     sdmmc_event_t unhandled_events = { 0 };
+    int t0 = esp_timer_get_time();
```

Comment:
The return type of esp_timer_get_time is int64_t, suggest using that for the variable type.

(applies to a bunch of other assignments, as well)